tsqa/conf_plugin.py

```diff
@@ -0,0 +1,28 @@
+import logging
```

Comment:
We might want to put these files into a "plugin" directory, instead of calling the file conf_plugin.py if this is going to he the way to add command line flags going forward, otherwise this directory is going to get cluttered :)